src/main/proto/spine/options.proto

```diff
@@ -1299,31 +1299,59 @@ message IfHasDuplicatesOption {
 //
 // For unbounded ranges, please use `(min)` and `(max) options.
 //
+// The option supports all Protobuf numeric fields.
+//
 message RangeOption {
 
     // The default error message.
     option (default_message) = "The field `${parent.type}.${field.path}` must be within"
-        " the following range: `${range.value}`.";
+        " the following range: `${range.value}`. The passed value: `${field.value}`.";
 
     // The string representation of the range.
     //
-    // The range can be open (not including the endpoint) or closed (including the endpoint) on
-    // each side. Open endpoints are indicated using a parenthesis (`(`, `)`). Closed endpoints are
-    // indicated using a square bracket (`[`, `]`).
+    // ## Syntax
+    //
+    // 1. The range can be open (not including the endpoint) or closed (including the endpoint)
```

Comment:
Here you should say, "There are two types of range borders", and then have an example for each type, along with the explanation.